Resources/public/js/views/cof-creationcontenttypeselectorview.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-contenttypeselectorview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the Content Creation Widget Content Type Selector View class
+     *
+     * @module cof-contenttypeselectorview
+     */
+    Y.namespace('cof');
+
+    var CLASS_TOOLTIP_HIDDEN = 'item-selected--tooltip-hidden',
+        SELECTOR_ITEM_SELECTED = '.ez-selection-filter-item-selected',
+        ATTR_ID = 'data-id',
+        ATTR_DESCRIPTION = 'data-description';
+
+    /**
+     * The Content Creation Widget Content Type Selector view
+     *
+     * @namespace cof
+     * @class ContentTypeSelectorView
+     * @constructor
+     * @extends eZ.ContentTypeSelectorView
+     */
+    Y.cof.ContentTypeSelectorView = Y.Base.create('ContentTypeSelectorView', Y.eZ.ContentTypeSelectorView, [], {
+        initializer: function () {
+            this.get('container').addClass(this._generateViewClassName(Y.eZ.ContentTypeSelectorView.NAME));
+
+            this.after('filterViewChange', this._attachOnSelectEventListener, this);
+        },
+
+        /**
+         * Attach event listener on select.
+         *
+         * @protected
+         * @method _attachOnSelectEventListener
+         */
+        _attachOnSelectEventListener: function () {
+            this.get('filterView').on('select', this._addDescriptionToSelectedItem, this);
+        },
+
+        /**
+         * Adds the description to the selected item.
+         *
+         * @protected
+         * @method _addDescriptionToSelectedItem
+         * @param event {Object} event facade
+         */
+        _addDescriptionToSelectedItem: function (event) {
+            var contentTypeItem = event.elementNode,
+                contentTypeItemId = contentTypeItem.getAttribute(ATTR_ID),
+                contentTypeGroups = this.get('contentTypeGroups');
+
+            contentTypeGroups.forEach(function (contentTypeGroup) {
+                contentTypeGroup.get('contentTypes').forEach(function (contentType) {
+                    if (contentTypeItemId === contentType.get('id')) {
+                        contentTypeItem.setAttribute(ATTR_DESCRIPTION, contentType.get('descriptions')[contentType.get('mainLanguageCode')]);
```

Comment:
What do you think about storing it in a variable? `contentType.get('descriptions')[contentType.get('mainLanguageCode')]`? It would improve the code readability.